src/python/grapl-common/grapl_common/grapl_tracer.py

```diff
@@ -0,0 +1,54 @@
+from opentelemetry import trace
+from opentelemetry.exporter.zipkin.json import ZipkinExporter
+from opentelemetry.instrumentation.botocore import BotocoreInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+def get_tracer(service_name: str, module_name: str) -> trace.Tracer:
+    """
+    This gets a tracer for instrumenting python apps manually with opentelemetry.
+
+    Usage
+    set OTEL_EXPORTER_ZIPKIN_ENDPOINT = "http://100.115.92.202:9411/api/v2/spans" as an environment variable
+
+    from grapl_common.grapl_tracer import get_tracer
+    ...
+    TRACER = get_tracer(service_name='provisioner', module_name=__name__)
+    ...
+    foo():
+        with TRACER.start_as_current_span("foo"):
+            ...
+
+    :param service_name: App/Service name ie provisioner
+    :param module_name: This should be __name__
+    :return: Tracer:
+    """
+    provider = TracerProvider(
+        resource=Resource.create(
+            {
+                "service.name": service_name,
+                # TODO add things like instance_id, etc here
```

Comment:
Will we eventually have some kind of schema for what kind of information we want to capture here? Or is there already some commonly-agreed-upon schema out in the wild that we could use (or are currently using in this code)?

Reply:
Yep. We'll be primarily using the opentracing schemas, although I could see us adding a few things that are grapl unique
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md